### DISCUSSION TITLE
Why data in minio transfers more quickly when use explorer to download？

### DISCUSSION THREAD

**Discussion by forestbat** [minio/minio#19071] [Q&A] [answered]
I'm developing projects for a university，and its minio server is deployed in education network，so when I use `python.minio` or `mc.exe` to download data to public network，its speed can’t bigger than 3MB/s.
However when I download data from minio console in explorer，strange things happened：

[![enter image description here][1]][1]

It's obvious that request in explorer had transferred only 245.33MB，however the file `gpm_gfs.nc` which I want to download is about **2.3GB**；total time cost is about 2.16min.

And about a month ago，downloading data from explorer's minio console will be catched by explorer's download manager，and its average speed is ~50MB/s. The explorer doesn't use other proxies.

So can anyone tell me why data transfers faster in explorer？And how can I optimize my code or network to break speed limitation of education network?

  [1]: https://i.stack.imgur.com/OfzPZ.png

**Comment by klauspost** [comment, marked as answer]
This is the first time anyone has complained that MinIO is too fast! :D

My guess is that the file is being gzip compressed when it is downloading. The S3 API does not do that - but the web API likely will.  It is furthermore disabled in mc for long reasons.

Good luck with your project otherwise!
